docs/cockpit.rst

```diff
@@ -23,29 +23,21 @@ the host you are running the ADB on.
 
 1. Start the ADB
 
-2. Log into the ADB
+2. Start cockpit
 
-   ``$ vagrant ssh``
+   ``$ vagrant service-manager restart cockpit``
```

Comment:
@budhrg I've added https://github.com/projectatomic/vagrant-service-manager/pull/218

It relies on the behavior of systemctl restart